src/app/page.tsx

```diff
@@ -15,14 +17,24 @@ const Home = () => {
 
   return (
     <>
+      <header className="w-[100%] bg-gray-700 p-4 text-left text-2xl">
+        HackRF.app
+      </header>
       <main className="flex h-full min-h-screen flex-col items-center justify-between p-1">
         <SerialLoader>
           <Controller />
         </SerialLoader>
       </main>
-      <footer className="text-center text-blue-500">
+      <footer className="text-center text-sm text-white">
+        <p className="pb-1">
+          Mayhem Project - {new Date().getFullYear()} - LICENSE
```

Comment:
@jLynx I need some input on this, do I add a link to a license file or just remove the license part?

Reply:
Lets just remove the license part for now, we can add it back at a later stage. Also if we can update the name to be MayhemHub rather than Mayhem Project